src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -199,6 +198,13 @@ private void checkCentralTenantExistsOrThrow() {
     }
   }
 
+  private void checkAdminUserIdPresentOrThrow(UUID adminUserId) {
+    if (Objects.isNull(adminUserId) || StringUtils.isBlank(adminUserId.toString())) {
+      log.warn("User id is null");
```

Comment:
Please specify more concrete message, for example the same as you using in exception message and also delete comma at the end